test/test-eltwise-reduce-mod.cpp

```diff
@@ -79,5 +79,20 @@ TEST(EltwiseReduceMod, 4_2) {
   CheckEqual(result, exp_out);
 }
 
+// https://github.com/intel/hexl/issues/86
+TEST(EltwiseReduceMod, LargeModError) {
```

Comment:
Maybe it can be called LargeModTest or LargeMod? unless you are expecting an error

Reply:
I removed it; the (EltwiseReduceModTest, Random) test was sufficient to reproduce the error.